cocos/spine/skeleton.ts

```diff
@@ -1569,20 +1569,23 @@ export class Skeleton extends UIRenderer {
         if (this.debugBones || this.debugSlots || this.debugMesh) {
             if (!this._debugRenderer) {
                 const debugDrawNode = new Node('DEBUG_DRAW_NODE');
+                debugDrawNode.layer = this.node.layer;
                 debugDrawNode.hideFlags |= CCObject.Flags.DontSave | CCObject.Flags.HideInHierarchy;
                 const debugDraw = debugDrawNode.addComponent(Graphics);
-                debugDraw.lineWidth = 1;
+                debugDraw.lineWidth = 5;
```

Comment:
The modification of lineWidth is based on what considerations?

Reply:
’cocos/spine/assembler/simple.ts‘ lineWidth allways is '5', I have a requirement to determine the lineWidth based on the worldScale of the animation. After this modification, the engine's logic remains unchanged, and I can specify the linewidth in the business logic.